### PR TITLE
Гусак Никита, 2537

### PR DESCRIPTION
**Pull request by pinguinson**

### CODE REVIEW

**Comment by thevery** [comment]
ok, but only ~3 FPS
I'll probably make the limit lower after reviewing all the submissions

File: app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java [hunk 1]
```diff
@@ -22,10 +24,14 @@
...
     Thread thread = null;
     volatile boolean running = false;
+    Paint paint = new Paint();
 
     public WhirlView(Context context) {
         super(context);
         holder = getHolder();
+        if (!isInEditMode()) {
+            setLayerType(SurfaceView.LAYER_TYPE_HARDWARE, null);
+        }
```
Comment (thevery) [inline app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java:34]: does this really helps?

**Comment by thevery** [comment]
reformatting code manually is a really bad idea when you are working in team

**Comment by thevery** [comment]
attempt to minimize redraws with `changed[x][y]` is nice though

File: app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java (hunk 1 above)
Comment (pinguinson) [inline app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java:34]: Android Dev site says it should enable hardware acceleration.
UPD: Just found this: "Hardware acceleration is enabled by default if your
Target API level is >=14", so these lines are unnecessary.

Comment (thevery) [inline app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java:34]: that was exactly I'm talking about

File: app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java [hunk 2]
```diff
@@ -60,40 +67,46 @@ public void run() {
 
     @Override
     public void onSizeChanged(int w, int h, int oldW, int oldH) {
-        width = w/scale;
-        height = h/scale;
+        width  = w / scale;
```
Comment (thevery) [inline app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java:70]: field size must be 320*240 scaled to fullscreen

**Comment by pinguinson** [comment]
Should i fix all issues and create a new pull request?

**Comment by thevery** [comment]
Yes since this one is already reviewed

**Pull request closed without merging**